src/Server.cpp

```diff
@@ -206,6 +206,24 @@ void Server::_purgeClient(const int& fd) {
     }
 }
 
+static std::string findCommands(std::string& fullMsg , std::string input) {
```

Comment:
You can subtract the one you find from the other one so it becomes more efficient and you don't have to use `find()` again. for example
```C++
        if ( firstNL < firstRNL ) {
            fullMsg += input.substr(0 , firstNL) + "\r\n";
            input.erase(0 , firstNL + 1);
            firstRNL -= firstNL;
            firstNL = input.find("\n", 1);
        }
```